src/main/java/com/coveo/pushclient/ApiStatus.java

```diff
@@ -1,9 +1,9 @@
-package com.coveo.pushapiclient;
+package com.coveo.pushclient;
 
 /**
- * Enum for possible PushAPI statuses. See [Updating the Status of a Push Source](https://docs.coveo.com/en/35).
+ * Enum for possible PushAPI statuses. See [Updating the Status of a Push SourceClient](https://docs.coveo.com/en/35).
  */
-public enum PushAPIStatus {
+public enum ApiStatus {
```

Comment:
Could `SourceStatus` be a better name for this enum? Is it the status of the api? or the source?

Reply:
yep i will double check and name it accordingly , thanks